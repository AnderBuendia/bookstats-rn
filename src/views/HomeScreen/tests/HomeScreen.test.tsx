import { Button, Text, TextInput, View } from 'react-native';
import { fireEvent, render, waitFor } from '@testing-library/react-native';

describe.skip('Test Home Screen', () => {
  it('Should show Bookstats title text', () => {});
});
