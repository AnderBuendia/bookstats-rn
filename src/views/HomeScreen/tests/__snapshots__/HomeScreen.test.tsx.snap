// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home Screen Should show Bookstats title text 1`] = `/Bookstats/i`;
